src/momento/aio/_scs_control_client.py

```diff
@@ -70,5 +80,58 @@ async def list_caches(self, next_token: Optional[str] = None) -> ListCachesRespo
         except Exception as e:
             raise _cache_service_errors_converter.convert(e)
 
+    async def create_signing_key(
+        self, ttl_minutes: int, endpoint: str
```

Comment:
I find it a little awkward to pass the `endpoint` as an arg here just to be re-packaged into `CreateSigningKeyResponse` and returned.

If you assign `self._endpoint = endpoint` in the ctor you can use that.

Reply:
This is the data plane endpoint, not the control plane endpoint. That's why I'm passing it differently here

Reply:
oh right!